tutorials/tour/anonymous-function-syntax.md

```diff
@@ -51,3 +51,33 @@ Function1[Int, Int]
 Function2[Int, Int, String]
 Function0[String]
 ```
+
+The following example shows how to use anonymous function of the beginning of this page
+
+```tut
+package tour
+
+object AnonymousFunction {
+
+  /**
+   * Method to increment an integer by one.
+   */
+  var anonymousIncrementFunction = (x: Int) => x + 1
+
+  /**
+   * Main method
+   * @param args application arguments
+   */
+  def main(args: Array[String]) {
+
+    // Create an integer to test the anonymous function with
+    var myInteger: Int = 0;
+
+    println(myInteger) // Prints: 0
+
+    myInteger = anonymousIncrementFunction(myInteger)
```

Comment:
You could do something like this to show that a previously incremented val will be incremented again:

```val myInteger = 0    

println(myInteger) // Prints: 0

val myIncrementedInteger = anonymousIncrementFunction(myInteger)

println(myIncrementedInteger) // Prints: 1

println(anonymousIncrementFunction(myIncrementedInteger)) // Prints: 2

println(anonymousIncrementFunction(anonymousIncrementFunction(myInteger)) // Prints: 2
```

It is more idiomatic to assign a result to a new val rather than reusing it.